Mathlib/LinearAlgebra/TensorProduct/Tower.lean

```diff
@@ -219,6 +219,62 @@ theorem map_smul_left (b : B) (f : M →ₗ[A] P) (g : N →ₗ[R] Q) : map (b 
   simp_rw [curry_apply, TensorProduct.curry_apply, restrictScalars_apply, smul_apply, map_tmul,
     smul_apply, smul_tmul']
 
+variable (A M) in
+/-- Heterobasic version of `LinearMap.lTensor` -/
+def lTensor (f : N →ₗ[R] Q) : M ⊗[R] N →ₗ[A] M ⊗[R] Q :=
+  map LinearMap.id f
```

Comment:
it might be easier to bundle it into `(N →ₗ[R] Q) →ₗ[R] M ⊗[R] N →ₗ[A] M ⊗[R] Q` straightaway so that you won't need to provide those add and sub lemmas.